python/tank_vendor/shotgun_deploy/manager.py

```diff
@@ -205,25 +205,15 @@ def bootstrap_engine(self, engine_name, entity=None):
         return engine
 
 
-    def _bootstrap_sgtk(self, engine_name, entity):
-        """
-        Create an sgtk instance for the given engine and entity.
-
-        If entity is None, the method will bootstrap into the site
-        config. This method will attempt to resolve the config according
-        to business logic set in the associated resolver class and based
-        on this launch a configuration. This may involve downloading new
-        apps from the toolkit app store and installing files on disk.
+    def get_config(self, engine_name, entity):
```

Comment:
By making this public, we are committing to the whole Configuration class as a public thing too. I suggest we keep it an internal thing until the need arises to make it public. Does that make sense? At least for the time being?